segm8/output_format.py

```diff
@@ -0,0 +1,14 @@
+"""This module describes constants that define the alignment methods and
+number systems used by SegM8 objects."""
+
+
+class Align:
+    """Constants describing the alignment method."""
+    RIGHT = 0
+    LEFT = 1
+
+
+class NumberSystem:
```

Comment:
IMHO "Radix"  is ​​shorter and quite unambiguously means the same thing. At your discretion.

Reply:
To be honest, this is the first time I've heard this word from you :) I asked three people in the office: one knows, two do not.
Perhaps I'll leave it that way.